### DISCUSSION TITLE
Movies

### DISCUSSION THREAD

**Discussion by hernessteve** [NAIST-SE/AttractingContributors#392] [General]
Where do you watch movies usually?
Netflix ? Some other services?

**Comment by ellielo** [comment]
Very easy - I have a firestick, Kodi and different movie apks.
There are tons of them [https://www.firesticktricks.com/best-movie-apks.html](https://www.firesticktricks.com/best-movie-apks.html). Cinema HD, Tubi TV, Media Box HD - all depends on your likings. Some of them are paid , another one can be used absolutely free of charge.
